src/Microsoft.Fx.Portability.Offline/Data.cs

```diff
@@ -92,7 +92,7 @@ private static Stream OpenFileOrResource(string path)
 
                 if (stream == null)
                 {
-                    throw new FileNotFoundException("Could not find data file next to or embedded in assembly.", path);
+                    throw new PortabilityAnalyzerException($"Could not find data file '{path}' next to or embedded in assembly.");
```

Comment:
Since this is a somewhat common error path, would it make sense to add a little more detail here about recommended next steps? Perhaps let the user know that the file should either be present when Microsoft.Fx.Portability.Offline is built or be placed next to the assembly?